app/stats/templates/base.html

```diff
@@ -1,32 +1,52 @@
+{% load django_htmx %}
+{% load tailwind_tags %}
 {% load static %}
 
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Stats page</title>
+        <title>The Green Web Foundation - Stats</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+
         <!-- Load Vega, Vega-Lite & Vega Embed -->
-        <script src="{% static 'assets/vega.min.js' %}"></script>
-        <script src="{% static 'assets/vega-lite.min.js' %}"></script>
-        <script src="{% static 'assets/vega-embed.min.js' %}"></script>
+        <script src="{% static 'javascript/assets/vega.min.js' %}"></script>
```

Comment:
Generally speaking, javascript is a kind of asset, like an image or CSS file is.

It would be better to have `assets/javascript` or even `assets/js`